app.py

```diff
@@ -54,18 +54,31 @@ def config():
         TWILIO_SYNC_SERVICE_SID=os.environ['TWILIO_SYNC_SERVICE_SID'],
     )
 
-@app.route('/token', methods=['POST', 'GET'])
-def token(identity = None):
+@app.route('/token', methods=['GET'])
+def randomToken():
+    return generateToken(fake.user_name())
+
+
+@app.route('/token', methods=['POST'])
+def createToken():
+    # Get the request json or form data
+    content = request.get_json() if request.get_json() else request.form
+    # get the identity from the request, or make one up
+    identity = content['identity'] if content['identity'] else fake.user_name()
```

Comment:
`identity = content.get('identity', fake.user_name()`

Reply:
Also fixed in a5e8a25
Thanks again!